lib/ansible/plugins/connection/nspawn.py

```diff
@@ -0,0 +1,152 @@
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import os
```

Comment:
Should import os.path here as well (it's an implementation detail that os.path works after only importing os.)